config/routes.rb

```diff
@@ -23,6 +23,11 @@
   # WEBSOCKETS
   mount ActionCable.server => '/cable'
 
+  # DELAYED JOB WEB
+  authenticated :user do
```

Comment:
It's [a devise helper](https://github.com/plataformatec/devise/wiki/How-To:-Define-resource-actions-that-require-authentication-using-routes.rb)